Classes/Systems/GithubAPIDateFormatter.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 
 private let dateFormatter = ISO8601DateFormatter()
 func GithubAPIDateFormatter() -> ISO8601DateFormatter {
+    // TODO: REVISE (Should be static)
     // https://developer.github.com/v3/#schema
     return dateFormatter
```

Comment:
Not sure if I'm going insane here, I can't remember if you have a let at root scope like this if it's the same as static? Might be fine, but we definitely shouldn't be creating multiple date formatters!

Reply:
Yup! Same as static. You can't static w/out some sort of namespacing (`enum`, `struct`, `class`). This only creates a single instance.

Reply:
Bit off topic, but recently saw something I really liked in our codebase at work, looking something like this:

```swift
enum A {
  static let
  hi = "hi",
  bye = "bye"
}
```

Using just "one" `static let`.